src/plugins/dropdownMenu/test/dropdownMenu.e2e.js

```diff
@@ -618,4 +618,187 @@ describe('DropdownMenu', function () {
       expect(keys).toEqual(['make_read_only', 'col_left']);
     });
   });
+
+  it('should be possible undo the alignment process by calling the \'Undo\' action without contextMenu', () => {
+    const hot = handsontable({
+      data: Handsontable.helper.createSpreadsheetData(9, 9),
+      dropdownMenu: true
+    });
+
+    // top 3 rows center
+    selectCell(0, 0, 2, 8);
+    hot.getPlugin('dropdownMenu').executeCommand('alignment:center');
+
+    // middle 3 rows unchanged - left
+
+    // bottom 3 rows right
+    selectCell(6, 0, 8, 8);
+    hot.getPlugin('dropdownMenu').executeCommand('alignment:right');
+
+    // left 3 columns - middle
+    selectCell(0, 0, 8, 2);
+    hot.getPlugin('dropdownMenu').executeCommand('alignment:middle');
+
+    // middle 3 columns unchanged - top
+
+    // right 3 columns - bottom
+    selectCell(0, 6, 8, 8);
+    hot.getPlugin('dropdownMenu').executeCommand('alignment:bottom');
+
+    var cellMeta = hot.getCellMeta(0, 0);
+    expect(cellMeta.className.indexOf('htCenter')).toBeGreaterThan(-1);
```

Comment:
Tests look good. However, it would be nice to have [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) instead of `.indexOf`. It would be more readable. The decision of change is up to you.